service/src/integrationTest/kotlin/fi/espoo/evaka/ChildBuilder.kt

```diff
@@ -0,0 +1,280 @@
+package fi.espoo.evaka
+
+import fi.espoo.evaka.application.ApplicationStatus
+import fi.espoo.evaka.daycare.service.AbsenceType
+import fi.espoo.evaka.daycare.service.CareType
+import fi.espoo.evaka.placement.PlacementType
+import fi.espoo.evaka.shared.db.Database
+import fi.espoo.evaka.shared.dev.DevAssistanceNeed
+import fi.espoo.evaka.shared.dev.DevChild
+import fi.espoo.evaka.shared.dev.DevPerson
+import fi.espoo.evaka.shared.dev.insertTestAbsence
+import fi.espoo.evaka.shared.dev.insertTestApplication
+import fi.espoo.evaka.shared.dev.insertTestAssistanceNeed
+import fi.espoo.evaka.shared.dev.insertTestBackUpCare
+import fi.espoo.evaka.shared.dev.insertTestChild
+import fi.espoo.evaka.shared.dev.insertTestDaycareGroupPlacement
+import fi.espoo.evaka.shared.dev.insertTestNewServiceNeed
+import fi.espoo.evaka.shared.dev.insertTestPerson
+import fi.espoo.evaka.shared.dev.insertTestPlacement
+import fi.espoo.evaka.shared.dev.insertTestPlacementPlan
+import fi.espoo.evaka.shared.domain.FiniteDateRange
+import java.lang.IllegalStateException
+import java.time.LocalDate
+import java.util.UUID
+
+class ChildBuilder(
+    private val tx: Database.Transaction,
+    private val today: LocalDate
+) {
+    fun childOfAge(years: Int, months: Int = 0, days: Int = 0): ChildFixture {
+        val dob = today.minusYears(years.toLong()).minusMonths(months.toLong()).minusDays(days.toLong())
+        val childId = tx.insertTestPerson(DevPerson(dateOfBirth = dob))
+        tx.insertTestChild(DevChild(childId))
+        return ChildFixture(tx, today, childId)
+    }
+
+    class ChildFixture(
```

Comment:
If the idea is to not use these classes directly (i.e. calling `ChildBuilder.ChildFixture(...)`) then the constructors shouldn't be public in my opinion.